core/messaging/__init__.py

```diff
@@ -127,6 +127,7 @@ class SharedMessages:
         "You entered {wrong_type} instead of a date. Check the cell is formatted as a date, "
         "for example, Dec-22 or Jun-23"
     )
+    GENERIC_WRONG_TYPE = "You entered {wrong_type} instead of {expected_type}. Check the data is correct."
```

Comment:
Just checking that this content has come from/been seen by @Kateharries? It looks good to me though, and seems to be consistent with other error messages.

Reply:
The change looks good to me. Thanks for flagging. I approve! :)